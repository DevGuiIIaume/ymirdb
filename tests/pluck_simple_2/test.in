set a 1 2 3
pluck a 2
get a
del a
pluck a 2
bye
