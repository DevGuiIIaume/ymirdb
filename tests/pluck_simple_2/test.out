> ok

> 2

> [1 3]

> ok

> no such key

> bye
